xpmem-lib.spec

```diff
@@ -5,7 +5,7 @@
 %global debug_package %{nil}
 
 Summary: XPMEM: Cross-partition memory
-Name: xpmem-lib
+Name: xpmem
```

Comment:
shall we also rename spec file to xpmem.spec?

Reply:
fixed

Reply:
Generally I agree. But do you want to have a single spec file? A separate spec file for the kernel (and if so: how to name it?)

Reply:
if we need to have two spec files, we could have `xpmem.spec` and `xpmem-kmod.spec`/`xpmem-module.spec`?

Reply:
@tzafrir-mellanox do you see an advantage for a separate spec file for kernel?